Application/EdFi.Ods.Features/OpenApiMetadata/Providers/OpenApiMetadataCacheProvider.cs

```diff
@@ -184,15 +198,26 @@ void AddToCache(IEnumerable<OpenApiContent> openApiContents)
                 foreach (var openApiContent in openApiContents)
                 {
                     // we want to force an update if the document has changed.
-                    _openApiMetadataMetadataCache
-                        .AddOrUpdate(
-                            $"{openApiContent.Section}-{openApiContent.Name}",
-                            openApiContent,
-                            (key, existing) => existing.GetHashCode() != openApiContent.GetHashCode()
-                                ? openApiContent
-                                : existing);
+                    newCache.AddOrUpdate(
+                        $"{openApiContent.Section}-{openApiContent.Name}",
```

Comment:
The change to the dictionary on 76 above, which results in it being case-sensitive, is causing cache misses when requesting some OpenApi endpoints for some features, including profiles and change queries. The cache misses result in 404 errors being returned when they are requested. An example endpoint where I see this is /profiles/sample-profile-resource-includeonly/swagger.json. 

If the cache dictionary needs to remain as it is, then changing this to be `"{openApiContent.Section}-{openApiContent.Name.ToLowerInvariant()}",` may be the most straightforward solution.

Reply:
Changed line 120 from this:

```csharp
var newCache = new ConcurrentDictionary<string, OpenApiContent>();
```
... to this...
```csharp
var newCache = new ConcurrentDictionary<string, OpenApiContent>(StringComparer.OrdinalIgnoreCase);
```